src/utils/webStorage.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  StateTypes,
+  StoreManagementAdapter,
+  WebStorageUnityType,
+} from "../typings";
+
+export default class WebStorageAdapter implements StoreManagementAdapter {
+  protected storage: Storage;
+
+  protected static stringifyValues(value: StateTypes) {
+    if (!value) return `${value}`;
+
+    if (typeof value === "object") {
+      const first = value.at(0);
+      if (first && typeof first !== "string") {
```

Comment:
Doesn't an empty string evals to false?

Reply:
No
![image](https://github.com/dede999/st-mary-state-manager/assets/7433785/60514110-f55b-4443-a58b-584e7ab5db8f)

Reply:
The check for an empty array was removed on commit 55a96ee4, because it was useless